lists-service/cicd/database/app/database.yaml

```diff
@@ -0,0 +1,129 @@
+AWSTemplateFormatVersion: '2010-09-09'
+
+Description: 'A DocumentDB Cluster with AutoScaling'
+
+Parameters:
+  pAdminUser:
+    Description: The username for the DocumentDB admin user
+    Type: String
+    Default: admin
+  pInstanceClass:
+    Description: The DocumentDB instance class
+    Type: String
+    Default: db.t4g.medium
+  pInstanceCount:
+    Description: The number of instances in the DocumentDB cluster
+    Type: Number
+    Default: 1
+  pVpcStackName:
+    Description: The name of the Bedrock VPC stack
+    Type: String
+  pBuild:
+    Description: The CodeBuild build number
+    Type: Number
+  pCleanBranch:
+    Description: The clean branch
+    Type: String
+  pProductName:
+    Description: The product name
+    Type: String
+  pEnvironment:
+    Description: Environment for this DocumentDB instance
+    AllowedValues:
+      - development
+      - develop
+      - staging
+      - production
+    Type: String
+
+Conditions:
+  IsDev: !Equals
+    - !Ref pEnvironment
+    - development
+  IsProd: !Equals
+    - !Ref pEnvironment
+    - production
+
+Resources:
+
+  DocDbSubnetGrp:
+    Type: AWS::DocDB::DBSubnetGroup
+    Properties:
+      DBSubnetGroupDescription: DocumentDB subnet group
+      SubnetIds: !Split
+        - ','
+        - Fn::ImportValue:
+            Fn::Sub: ${pVpcStackName}-PriSubnets
+      Tags:
+        - Key: Name
+          Value: Lists DocumentDB SubnetGroup
```

Comment:
For resource name tagging the resource type can be left off as we already know what it is. The Name should be a unique name for environment/branch so when you're looking at a list of resources you know where it belongs, you can use something like

```
Value: !Sub
    - ${pProductName}-$ResourceName}
    - !If [ IsDev, !Ref pCleanBranch, !Ref pEnvironment ]
```